vyper/compile_lll.py

```diff
@@ -105,8 +105,6 @@ def compile_to_assembly(code, withargs=None, break_dest=None, height=0):
     elif code.value == 'repeat':
         o = []
         loops = num_to_bytearray(code.args[2].value)
-        if not loops:
-            raise Exception("Number of times repeated must be a constant nonzero positive integer: %r" % loops)
```

Comment:
You were able to find that there was no scenario where `loops` would not be set?

Reply:
It's a duplicate exception, so never gets reached ;) in parser_utils.py line 121 the case is already cover.